### PR TITLE
[LocalNet] Add observability stack

### PR DESCRIPTION
**Pull request by okdas**
## Summary

Brings observability stack to LocalNet.

## Issue

- #458 

## Type of change

Select one or more:

- [x] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR. **THIS IS VERY EXPENSIVE**, so only do it after all the reviews are complete.
- [x] **Documentation changes**: `make docusaurus_start`

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have commented my code
- [ ] I have performed a self-review of my own code; both comments & source code
- [ ] I create and referenced any new tickets, if applicable
- [ ] I have left TODOs throughout the codebase, if applicable

### CODE REVIEW

**Review by Olshansk — APPROVED** [review]
I didn't see the data locally (yet) but did manage to access the dashboard so it's a great first step and good enough to merge in!

File: localnet/kubernetes/observability-loki-stack.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+loki:
+  isDefault: false
```
[the remark this answers is not in the document]
Comment (okdas) [reply localnet/kubernetes/observability-loki-stack.yaml:2]: @Olshansk no, - Prometheus is configured to be the default data source. Can't have two default data sources.

**Comment by Olshansk** [comment]
The RelayMiner showed some data! Any idea why the validator hasn't yet?


Screenshot 2024-04-10 at 5 05 07 PM
Screenshot 2024-04-10 at 5 05 03 PM

**Comment by okdas** [comment]
> Any idea why the validator hasn't yet?

Do you have this turned on or off in your config?

```
helm_chart_local_repo:
  enabled: true
```

If it is enabled, you'd need to pull in the latest changes in the chart repo. You usually don't need to turn on that feature unless you actively develop a helm chart. If it is disabled, and you still experience that issue, please let me know - we can take a look. I just double-checked and it worked on my machine.

**Pull request merged**